google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -93,6 +93,8 @@ def pyarrow_numeric():
 
 
 def pyarrow_bignumeric():
+    # 77th digit is partial.
+    # https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#decimal_type
```

Comment:
```suggestion
    # https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#decimal_types
```

The anchor in the URL seems incorrect, it just positions you at the top of the page.